ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -169,14 +169,17 @@ public struct Avatar: View {
         let shouldDisplayPresence = presence != .none
         let isRingVisible = state.isRingVisible
         let hasRingInnerGap = state.hasRingInnerGap
+        let ringThicknessToken: CGFloat = tokens.ringThickness
```

Comment:
compiler couldn't infer cgfloat?

Reply:
It interprets the token value as an optional, and explicitly defining type removed the error